java/test/jmri/jmrix/loconet/LnThrottleManagerTest.java

```diff
@@ -1179,16 +1164,31 @@ public void notifyDecisionRequired(jmri.LocoAddress address, DecisionType questi
     @BeforeEach
     public void setUp() {
         JUnitUtil.setUp();
-        memo = new LocoNetSystemConnectionMemo();
+        memo = new LocoNetSystemConnectionMemo(){
+            @Override
+            public LnPredefinedMeters getPredefinedMeters(){
+                return null;
+            }
+            @Override
+            public LnSensorManager getSensorManager(){
+                return null;
+            }
+            @Override
+            public LnTurnoutManager getTurnoutManager() {
+                return null;
+            }
+            @Override
+            public LnPowerManager getPowerManager() {
+                return null;
+            }
+        };
         lnis = new LocoNetInterfaceScaffold(memo);
         memo.setLnTrafficController(lnis);
         memo.configureCommandStation(LnCommandStationType.COMMAND_STATION_DCS100, false, false, false);
         memo.configureManagers();
-        tm = new LnThrottleManager(memo);
-        log.debug("new throttle manager is {}", tm.toString());
-        memo.getSensorManager().dispose(); // get rid of sensor manager to prevent it from sending interrogation messages
-        memo.getPowerManager().dispose(); // get rid of power manager to prevent it from sending slot 0 read message
-        memo.getPredefinedMeters().dispose(); // get rid of meter to prevent it reading slots
```

Comment:
It would be good to add these comments to the creation of the memo on line 1167. For example on line 1173:
```
            public LnSensorManager getSensorManager(){
                // don't create a sensor manager to prevent it from sending interrogation messages
                return null;
            }
```

Reply:
Thanks, have used the comments in the new memo creation.